### ISSUE TITLE
Updating cart quantities doesn't update #cartItemsTotal properly

### ISSUE THREAD

**Issue by tylerforesthauser** [lightspeedretail/webstore#664]
When viewing the Edit Cart modal, updating the quantity of line items adjusts the value of the #cartItemsTotal element incorrectly. Instead of returning the total number of items in the cart, it returns the number of line items.

core/protected/extensions/wsadvcheckout/assets/checkout.js (line 660):
```
$('#cartItemsTotal').text(updateResponse.shoppingCart.cartItems.length);
```

Reproduce by adding two separate items to your cart. Then view the cart and adjust the quantity of one of the items from 1 to 2. The total item count should be 3 but the value of #cartItemsTotal will remain at 2.

**Comment by gabeguz** [comment]
Thanks for the detailed report @tylerforesthauser -- I'll take a look sometime this week.

**Comment by jsfehler** [comment]
This issue has been logged in our bug tracker as WS-4418

**Comment by jsfehler** [comment]
This issue has been fixed in version 3.3.0.
